webpack.config-preprocessor-directives.js

```diff
@@ -29,6 +29,7 @@ const nodeModuleRelativeUrl = (isPackaging === "1") ?
 
 const data = {
     __APP_VERSION__: JSON.stringify(version),
+    __APP_NAME__: JSON.stringify(build.productName),
```

Comment:
Please update the PR's branch to sync with `upstream/develop`, see: https://github.com/readium/readium-desktop/commit/de8e69b2513c67adacf5b4b3bf3dcdf8cb2132ee

Reply:
done